packages/web/tests/components/Contributions/ContributionsBox.test.tsx

```diff
@@ -0,0 +1,22 @@
+import React from 'react';
+import { render, screen } from '../../testUtils/testTools';
+import { ContributionsBox } from '../../../src/components/Contributions';
+import { ContributionsList } from '../../../src/pages/app/contributions';
+// Mock repo objects
+const clist1: ContributionsList = {
+  id: '1',
+  nodeID: 'PR_00000',
+  description: 'clist1',
+  type: 'CLOSED',
+  score: 1,
+  contributedAt: new Date('1612-04-20'),
+  url: 'abc.com',
+};
+
+describe('Contributions Box', () => {
+  it('renders', async () => {
```

Comment:
Checking to see if it renders is not sufficient. Change this description and add more meaningful evaluations to the test below (like whether the fields within a contribution box render as expected and whether the link is clickable)